src/Heading.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+const Heading = () => {
```

Comment:
Good job extracting the heading!

Some small suggestions; 
- The most common convention for naming css classes is to use all lower case and have - between words. I would change the class name you use here to `app-header` to follow the convention.
- On line 10-11 you have styling logic set on the image directly. You should try to move all styling logic into CSS and in your JSX/HTML just add classes to indicate what styles should apply. So in this case, I would create a new CSS class and then add a the class on the `img` tag.